packages/block-editor/src/store/reducer.js

```diff
@@ -656,6 +809,8 @@ export const blocks = flow(
 
 		return state;
 	},
+
+	selection: blockSelection,
```

Comment:
This is a substantial change on the block-editor state, personally, I'm not seeing potential problems with this change but it would be good to get other people thoughts. cc: @youknowriad, @aduth

Reply:
All this turned out to be not necessary, so I restored it all. :)